$NetBSD: patch-ab,v 1.1 2006/11/28 18:57:25 rillig Exp $

SunPro cannot handle function-like macros on the command line.

--- configure.orig	2005-11-05 19:16:48.000000000 +0100
+++ configure	2006-11-28 19:28:51.822140712 +0100
@@ -23898,13 +23898,9 @@ echo $ECHO_N "checking whether the Xfce 
     echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6
     xdt_cv_CFLAGS="$xdt_cv_CFLAGS -DXFCE_PANEL_THREADED=1"
-    xdt_cv_CFLAGS="$xdt_cv_CFLAGS -DXFCE_PANEL_LOCK\(\)=gdk_threads_enter\(\)"
-    xdt_cv_CFLAGS="$xdt_cv_CFLAGS -DXFCE_PANEL_UNLOCK\(\)=gdk_threads_leave\(\)"
   else
     echo "$as_me:$LINENO: result: no" >&5
 echo "${ECHO_T}no" >&6
-    xdt_cv_CFLAGS="$xdt_cv_CFLAGS -DXFCE_PANEL_LOCK\(\)=do{}while\(0\)"
-    xdt_cv_CFLAGS="$xdt_cv_CFLAGS -DXFCE_PANEL_UNLOCK\(\)=do{}while\(0\)"
   fi
   XFCE4_PANEL_CFLAGS="$xdt_cv_CFLAGS"
 
